internal/services/services.go

```diff
@@ -0,0 +1,26 @@
+package services
+
+import (
+	"fmt"
+
+	"github.com/broadinstitute/sherlock/internal/db"
+)
+
+// Service is the data structure representing an indvidual applicaiton
+type Service struct {
+	ID        int    `json:"id,omitempty" db:"id"`
```

Comment:
Can this ever be empty?

Reply:
When POSTing a new service yes, when listing existing ones no.